app.js

```diff
@@ -26,8 +25,10 @@ openevent.config(['$urlRouterProvider', '$httpProvider', function($urlRouterProv
     //delete $httpProvider.defaults.headers.common['X-Requested-With'];
     $urlRouterProvider.otherwise('sessions');
     }]);
+
+var totalDays;
```

Comment:
You can have a module for it? I believe you can wrap it up in a sessions
module, like the way you access the API factory.

On Sun, Aug 16, 2015 at 11:02 PM, Arnav Gupta <notifications@github.com>
wrote:

> In app.js
> <https://github.com/fossasia/open-event-webapp/pull/61#discussion_r37146691>
> :
>
> > @@ -26,8 +25,10 @@ openevent.config(['$urlRouterProvider', '$httpProvider', function($urlRouterProv
> >      //delete $httpProvider.defaults.headers.common['X-Requested-With'];
> >      $urlRouterProvider.otherwise('sessions');
> >      }]);
> > +
> > +var totalDays;
>
> Because I am using it in app.js and also again in SessionController.js }
> What is the best way to have a global js variable ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fossasia/open-event-webapp/pull/61/files#r37146691>.
>



-- 
Mohit Singh KANWAL (Mr)

[image: Facebook] <http://facebook.com/mohitkanwal>  [image: Twitter]
<http://twitter.com/mohitkanwal>